CHARTER.md

```diff
@@ -1,107 +1,124 @@
+- d. The TSC may create, change, modify, or remove roles or their definitions, so long as the definitions of roles for the WG are publicly available in the WG repository.
 
-   - iii. A Contributor may become a Collaborator by a majority approval of the existing Collaborators. A Collaborator may be removed by a majority approval of the other existing Collaborators.
+- e. The TSC will be responsible for all aspects of oversight relating to the WG, which may include:
 
-   - iv. Maintainers are the initial Collaborators defined at the creation of the Technical Initiative. The Maintainers will determine the process for selecting future Maintainers. A Maintainer may be removed by two-thirds approval of the other existing Maintainers, or a majority of the other existing Collaborators.
+  - i. coordinating the direction of the WG;
+  - ii. approving, organizing or removing activities;
+  - iii. establish community norms, workflows, processes, release requirements, and templates for the operation of the WG;
+  - iv. appointing representatives to work with other open source or open standards communities;
+  - v. approving and implementing policies and processes for contributing (to be published in the WG repository) and coordinating with the Linux Foundation to resolve matters or concerns that may arise as set forth in Section 6 of this Technical Charter;
+  - vi. facilitating discussions, seeking consensus, and where necessary, voting on technical matters relating to the WG; and
+  - vii. coordinating any communications regarding the WG.
 
-- d. Participation in the Technical Initiative through becoming a Contributor, Collaborator, or Maintainer is open to anyone, whether a OpenSSF member or not, so long as they abide by the terms of this Technical Charter. 
+- f. The TSC may elect a TSC Chair, who will preside over meetings of the TSC and will serve until their resignation or replacement by the TSC. The TSC Chair will serve as the WG’s voting representative on the TAC. In the event that a proxy is requested by the TSC Chair, it may be any other TSC member so designated by a silent consensus of the TSC.
 
-- e. The TSC may create, change, modify, or remove roles or their definitions, so long as the definitions of roles for the Technical Initiative are publicly available in the Technical Initiative repository.
+- g. The current TSC Chair is:
 
-- f. The TSC may elect a TSC Chair, who will preside over meetings of the TSC and will serve until their resignation or replacement by the TSC.  **The TSC Chair, or any other TSC member so designated by the TSC, will serve as the Technical Initiative’s voting representative on the OpenSSF’s Technical Advisory Council (the "TAC").
+  - Eddie Knight, Sonatype (@eddie-knight)
```

Comment:
Given this is a charter, is it expected that the charter of the WG is modified to reflect the current TSC chair after each election cycle? or is this intended to be moved elsewhere (for example the repo README to indicate named roles)?

Reply:
My hope is to have a single source of truth for governance discussions. This doesn't need to be here, and I'm not sticking to it. 

But I've been part of too many groups that never reference the charter because information relevant to regular activity is maintained elsewhere. I'd like to encourage folks to come back to this document, if we can manage it.

Reply:
That makes sense then!